packages/catalog-readmodel-writer/src/model/utils.ts

```diff
@@ -0,0 +1,9 @@
+// Temporary workaround
+export const safeParserDate = (date: bigint | undefined): Date => {
```

Comment:
I know it's not introduced in this PR, but the name of this function has a typo :) 
You could rename it in `safeParseDate` so that it is a verb

Reply:
Also, maybe a more philosophical comment: it is `safe` if it throws? Maybe we could call it `parseDateOrThrow` or something similar